feature-detects/forms/inputnumber-l10n.js

```diff
@@ -31,7 +31,7 @@ define(['Modernizr', 'createElement', 'getBody', 'test/inputtypes', 'test/forms/
 
     body.insertBefore(div, firstChild);
 
-    div.innerHTML = '<input type="number" value="1.0" step="0.1"/>';
+    div.innerHTML = '<input type="number" value="1.0" step="0.1" style="position: fixed;" />';
```

Comment:
Maybe add also a `;top:0;`` like in https://github.com/Modernizr/Modernizr/blob/25f2f3d0030a828f03b533f47a61da514e2e5139/feature-detects/css/hyphens.js#L151 
?